src/main/com/example005/radek/printDiagonalsApplication.java

```diff
@@ -0,0 +1,57 @@
+package main.com.example005.radek;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+enum DiagonalType {
```

Comment:
Enum bude novy file. Neni to spatne, ale udelejme to cistejsi, i kdzy to nicemu nevadi.

Reply:
Přehozeno do nového souboru class souboru nenašel jsem, že by to mělo být v něčem jiném. Tak jen prosím o potvrzení nebo vyvrácení